src/main/java/de/doubleslash/keeptime/view/ManageWorkController.java

```diff
@@ -0,0 +1,260 @@
+      projectComboBox.getItems().addAll(model.getAvailableProjects());
+
+      // Dropdown Options
+      projectComboBox.setCellFactory(new Callback<ListView<Project>, ListCell<Project>>() {
+
+         @Override
+         public ListCell<Project> call(final ListView<Project> l) {
+            return new ListCell<Project>() {
+
+               @Override
+               protected void updateItem(final Project item, final boolean empty) {
+                  super.updateItem(item, empty);
+                  if (item == null || empty) {
+                     setGraphic(null);
+                  } else {
+                     final Color color = item.getColor();
+                     final String hexColor = String.format("#%02X%02X%02X", (int) (color.getRed() * 255),
+                           (int) (color.getGreen() * 255), (int) (color.getBlue() * 255));
+                     setStyle(" -fx-background-color: " + hexColor);
+                     setText(item.getName());
+
+                  }
+               }
+            };
+         }
+      });
+
+      // selected Item
+      projectComboBox.buttonCellProperty().bind(Bindings.createObjectBinding(() -> {
+
+         // Get the arrow button of the combo-box
+         final StackPane arrowButton = (StackPane) projectComboBox.lookup(".arrow-button");
+
+         return new ListCell<Project>() {
+
+            @Override
+            protected void updateItem(final Project item, final boolean empty) {
+               super.updateItem(item, empty);
+               if (empty || item == null) {
+                  setGraphic(null);
+               } else {
+                  final Color color = item.getColor();
+                  final String hexColor = String.format("#%02X%02X%02X", (int) (color.getRed() * 255),
+                        (int) (color.getGreen() * 255), (int) (color.getBlue() * 255));
+                  setStyle(" -fx-background-color: " + hexColor);
+                  setText(item.getName());
+               }
+
+               // Set the background of the arrow also
+               if (arrowButton != null) {
+                  arrowButton.setBackground(getBackground());
+               }
+            }
+
+         };
+      }, projectComboBox.valueProperty()));
+
+      // selected value showed in combo box
+      projectComboBox.setConverter(new StringConverter<Project>() {
+         @Override
+         public String toString(final Project project) {
+            if (project == null) {
+               return null;
+            } else {
+               return project.getName();
+            }
+         }
+
+         @Override
+         public Project fromString(final String string) {
+            return null;
+         }
+      });
+
+      projectComboBox.getSelectionModel().select(work.getProject());
+      final Color color = work.getProject().getColor();
+      final String hexColor = String.format("#%02X%02X%02X", (int) (color.getRed() * 255),
```

Comment:
duplicated code 3 times. In general we should eliminate duplicate code by extracting into a method :) But see comment above.